kobotoolbox/kpi
# [TASK-1167] Expose Team or Organization wording to front end

## Checklist

1. [x] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [x] Ensure the tests pass
4. [x] Run `./python-format.sh` to make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/main/CONTRIBUTING.md)
5. [x] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes
8. [x] Create a testing plan for the reviewer and add it to the Testing section

## Description

This PR adds a configurable setting to the Django admin interface that allows servers without Stripe enabled to toggle between the use of wording "Team" or "Organization" in the UI. When the setting is enabled, "Team" will be used, and when disabled, "Organization" will be used.

The setting is exposed to the `/environment` as `use_team_label: true/false`, which will be reflected when Stripe is not enabled.

## Notes

- Added a new `USE_TEAM_LABEL` setting to `CONSTANCE_CONFIG`, allowing administrators to configure whether "Team" or "Organization" is displayed on the interface.
- The setting is visible in the Django admin interface via the constance config, making it easily adjustable without code changes.
- This setting is also exposed in the `/environment` as `use_team_label: true/false`, enabling the frontend to adjust the terminology dynamically.
- The `/environment` will return this value when Stripe is not enabled, ensuring consistent terminology across the interface.

## Testing

1. Visit the Django admin panel.
2. Navigate to the Constance config section.
3. Locate the `USE_TEAM_LABEL` setting.
4. Toggle the value between `True` (for "Team") and `False` (for "Organization").
5. Save the changes.
6. Hit the `/environment` API and observe the value of `use_team_label` in the response.
